content/docs/09-registries-and-packs/4.5-create-pack.md

```diff
@@ -0,0 +1,718 @@
+
+<br /> <br />
+
+### Section: Nodes config
+In the **Nodes config** section, provide the details for the master and the worker pool. For learning purposes, you can use the following configuration:
+
+|**Field** | **Value for the master-pool**| **Value for the worker-pool**|
+|---| --- | ---|
+|Node pool name| *master-pool* | *worker-pool* |
+|Number of nodes in the pool| `1` | `1` |
+|Allow worker capability| *Checked* | Not applicable |
+|Enable Autoscaler | Not applicable | No |
+|Rolling update |  Not applicable | *Expand* <br /> Launch a new node first, then shut down the old one. |
+|Instance Type | General purpose `m4.large` | Copy from Master pool | 
+|Availability zones | **us-east-2a** <br />Choose only one availability zone. | Copy from Master pool | 
+|Disk size | 60 GiB | Copy from Master pool | 
+
+<br /> <br />
+
+### Section: Settings and Review
+Use the default values in the **Settings** section and review all configurations in the **Review** section.  If everything looks good, finish creating the cluster configuration. It will start deploying the cluster and may take up to 15-20 minutes to finish.
+
+<br />
+
+![Visual of the cluster health.](/tutorials/create-pack/registries-and-packs_create-pack_cluster-health.png)
+
+<br /> <br />
+
+You can also look at real-time metrics on the **Overview** tab, such as CPU and memory consumption. 
+
+<br /> <br />
+
+## Validate
+
+After the Palette finishes provisioning the cluster, the status will turn to **Running** and **Healthy**. You can access the application on the exposed port 8080 for the **Hello Universe** service. The application will look similar to the visual below. 
+
+
+<br />
+
+![Visual of the successful accessing the application using the load balancer URL.](/tutorials/create-pack/registries-and-packs_create-pack_success.png)
+
+<br /> <br />
+
+Congratulations, you've successfully deployed the *Hello Universe* application to the cluster. 
+
+<br /> <br />
+
+## Cleanup
+
+Delete the cluster and the profile by navigating to the **Cluster** and **Profile** sections in the left **Main Menu**, respectively. 
+
+<br />
+
+![Visual of deleting the cluster in the Palette console.](/tutorials/create-pack/registries-and-packs_create-pack_delete-cluster.png)
+
+<br /> <br />
+
+Delete the profile after deleting the cluster. 
+<br />
+
+
+![Visual of deleting the profile in the Palette console.](/tutorials/create-pack/registries-and-packs_create-pack_delete-profile.png)
+
+<br /> <br />
+
+Wait for the resources to clean up and ensure the successful resource deletion in the Palette console. 
```

Comment:
What about the local container? How do they clean that up? They should also remove the registry server.